drivers/media/i2c/s5k2xx.c

```diff
@@ -0,0 +1,1468 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
`GPL-2.0` should be replaced by `GPL-2.0-only`

Reply:
This should be modified in the source repo (msm8953-mainline/linux)